fastify.js

```diff
@@ -56,7 +57,22 @@ function build () {
       throw new Error(`${opts.method} method is not supported!`)
     }
 
-    opts[schema] = fastJsonStringify(opts.schema.out)
+    // Missing handler => schema is a function
+    if (!opts.handler && typeof opts.schema === 'function') {
+      opts.handler = opts.schema
+      opts.schema = {}
+    }
```

Comment:
This is not ok. This block should go within the get/post/put etc code. This does the same job, but it is unreadable, and it might allow further bugs down the road.